app/helpers/patient_query_helper.rb

```diff
@@ -159,35 +161,35 @@ def sort(patients, order, direction)
     when 'transferred_to'
       patients = patients.includes(:jurisdiction).order('jurisdictions.path ' + dir)
     when 'assigned_user'
-      patients = patients.order('CASE WHEN assigned_user IS NULL THEN 1 ELSE 0 END, assigned_user ' + dir)
+      patients = patients.order(Arel.sql('CASE WHEN assigned_user IS NULL THEN 1 ELSE 0 END, assigned_user ' + dir))
     when 'state_local_id'
-      patients = patients.order('CASE WHEN user_defined_id_statelocal IS NULL THEN 1 ELSE 0 END, user_defined_id_statelocal ' + dir)
+      patients = patients.order(Arel.sql('CASE WHEN user_defined_id_statelocal IS NULL THEN 1 ELSE 0 END, user_defined_id_statelocal ' + dir))
     when 'dob'
-      patients = patients.order('CASE WHEN date_of_birth IS NULL THEN 1 ELSE 0 END, date_of_birth ' + dir)
+      patients = patients.order(Arel.sql('CASE WHEN date_of_birth IS NULL THEN 1 ELSE 0 END, date_of_birth ' + dir))
     when 'end_of_monitoring'
-      patients = patients.order('CASE WHEN continuous_exposure = 1 THEN 1 ELSE 0 END,
-                                 CASE WHEN last_date_of_exposure IS NULL THEN patients.created_at ELSE last_date_of_exposure END ' + dir)
+      patients = patients.order(Arel.sql('CASE WHEN continuous_exposure = 1 THEN 1 ELSE 0 END,
+                                 CASE WHEN last_date_of_exposure IS NULL THEN patients.created_at ELSE last_date_of_exposure END ' + dir))
     when 'extended_isolation'
-      patients = patients.order('CASE WHEN extended_isolation IS NULL THEN 1 ELSE 0 END, extended_isolation ' + dir)
+      patients = patients.order(Arel.sql('CASE WHEN extended_isolation IS NULL THEN 1 ELSE 0 END, extended_isolation ' + dir))
     when 'symptom_onset'
-      patients = patients.order('CASE WHEN symptom_onset IS NULL THEN 1 ELSE 0 END, symptom_onset ' + dir)
+      patients = patients.order(Arel.sql('CASE WHEN symptom_onset IS NULL THEN 1 ELSE 0 END, symptom_onset ' + dir))
     when 'risk_level'
       patients = patients.order_by_risk(asc: dir == 'asc')
     when 'monitoring_plan'
-      patients = patients.order('CASE WHEN monitoring_plan IS NULL THEN 1 ELSE 0 END, monitoring_plan ' + dir)
+      patients = patients.order(Arel.sql('monitoring_plan IS NULL, collection_id ' + dir))
```

Comment:
I don't follow this change - what is `collection_id?`

Reply:
I don't think that was intentional - I made the correct change